lib/matplotlib/tests/test_colors.py

```diff
@@ -1301,6 +1301,62 @@ def test_to_rgba_array_alpha_array():
     assert_array_equal(c[:, 3], alpha)
 
 
+@pytest.mark.parametrize('rgba_alpha, alpha',
+                         [(('black', 1), 0.5), (('#000000', 0.5), None)])
+def test_to_rgba_array_color_alpha_tuple(rgba_alpha, alpha):
+    """to_rgba_array() accepts tuple (color, alpha) argument."""
+    booleans = mcolors.to_rgba_array(rgba_alpha, alpha=alpha) \
+               == mcolors.to_rgba_array('black', alpha=0.5)
+    assert booleans.all()
```

Comment:
I think these can be written more clear as:

```suggestion
def test_to_rgba_array_accepts_color_alpha_tuple():
    assert_array_equal(
        mcolors.to_rgba_array(('black', 0.9)),
        [[0, 0, 0, 0.9]])


def test_to_rgba_array_exlicit_alpha_takes_precedence_over_tuple_alpha():
    assert_array_equal(
        mcolors.to_rgba_array(('#000000', 0.9), alpha=0.5),
        [[0, 0, 0, 0.5]])
```